hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/clients/DefaultClientRegistry.java

```diff
@@ -0,0 +1,120 @@
+package org.hypertrace.traceenricher.enrichment.clients;
+
+import com.typesafe.config.Config;
+import io.grpc.Channel;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import org.hypertrace.core.attribute.service.cachingclient.CachingAttributeClient;
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.datamodel.StructuredTrace;
+import org.hypertrace.entity.data.service.client.EdsCacheClient;
+import org.hypertrace.entity.data.service.client.EntityDataServiceClient;
+import org.hypertrace.entity.data.service.rxclient.EntityDataClient;
+import org.hypertrace.entity.service.client.config.EntityServiceClientConfig;
+import org.hypertrace.entity.type.service.rxclient.EntityTypeClient;
+import org.hypertrace.trace.reader.attributes.TraceAttributeReader;
+import org.hypertrace.trace.reader.attributes.TraceAttributeReaderFactory;
+import org.hypertrace.trace.reader.entities.TraceEntityClientContext;
+import org.hypertrace.trace.reader.entities.TraceEntityReader;
+import org.hypertrace.trace.reader.entities.TraceEntityReaderFactory;
+import org.hypertrace.traceenricher.enrichment.enrichers.cache.EntityCache;
+
+public class DefaultClientRegistry implements ClientRegistry {
```

Comment:
Didn't seem worth testing this class as it's just client building in the constructor

Reply:
But we love big coverage numbers ;)

Reply:
Yeah, yeah me too. Not only are these statics a big pain to test, but they don't provide value since we've got no logic in here - just invocations to duplicate in the test.